torch/_inductor/codegen/triton.py

```diff
@@ -1291,7 +1309,10 @@ def load(self, name: str, index: sympy.Expr):
                 ep = ", eviction_policy='evict_first'"
         else:
             ep = ""
-        if (has_tmpmask or has_rindex) and indexing.has_mask():
+
+        if self._load_other:
+            other = f", other={constant_repr(self._load_other)}"
+        elif (has_tmpmask or has_rindex) and indexing.has_mask():
```

Comment:
I guess this is fine because we should always have a tmpmask when `self._load_other` is set, but in general this string should only be set inside this branch.